README.md

```diff
@@ -4,11 +4,7 @@
 
 # Zeitgeist: An Evolving Blockchain for Prediction Markets and Futarchy
 
-![Rust](https://github.com/zeitgeistpm/zeitgeist/workflows/Rust/badge.svg)
-
-<a href="https://t.me/zeitgeist_official">
-  <img src="https://img.shields.io/badge/telegram-https%3A%2F%2Ft.me%2Fzeitgeist__official-blue" />
-</a>
+![Rust](https://github.com/zeitgeistpm/zeitgeist/workflows/Rust/badge.svg) [![Codecov](https://codecov.io/gh/zeitgeistpm/zeitgeist/branch/main/graph/badge.svg?token=51WA0AHGWV)](https://codecov.io/gh/zeitgeistpm/zeitgeist) [![Discord](https://img.shields.io/badge/discord-https%3A%2F%2Fdiscord.gg%2FMD3TbH3ctv-purple)](https://discord.gg/MD3TbH3ctv) [![Telegram](https://img.shields.io/badge/telegram-https%3A%2F%2Ft.me%2Fzeitgeist__official-blue)](https://t.me/zeitgeist_official)
```

Comment:
> It is not considered particularly dangerous to have this token in a public repository. However, It's still not recommended to share it publicly as it could be used by someone to access the Codecov reports for the repository, but not access to the repository itself.
Additionally, if the token is compromised, someone could potentially use it to forge coverage reports to look better than it is.

*ChatGPT

Reply:
https://github.com/zeitgeistpm/zeitgeist/pull/950/commits/ac314accc18ac8ccf596f5f332ce440a5170d214

Reply:
You could also run `prettier -w README.md`. Not sure how adamant we want to be with this stuff.